packages/server/api/routes/users.router.js

```diff
@@ -27,4 +27,47 @@ router.get('/', (req, res, next) => {
     .catch(next);
 });
 
+/**
+ * @swagger
+ * /users:
+ *  post:
+ *    tags:
+ *    - users
+ *    summary: Create a user
+ *    description:
+ *      Will create a favorite.
+ *    produces: application/json
+ *    parameters:
+ *      - in: body
+ *        name: users
+ *        description: The user to create.
+ *        schema:
+ *          type: object
+ *          required:
+ *            - full_name
+ *            - email
+ *            - created_at
+ *            - uid
+ *          properties:
+ *            full_name:
+ *              type: string
+ *            email:
+ *              type: string
+ *            created_at:
+ *              type: date/time
+ *            uid:
+ *              type: date/time
+ *    responses:
+ *      201:
```

Comment:
Does it actually return a 201 response?

Reply:
changed to 200, or should it be removed?

Reply:
It's fine with 200